TableSort.svelte

```diff
@@ -57,19 +59,20 @@
                 sortOrder = [[fieldName, 0]]
             }
         }
-        th.className =
-            CLASSNAME_SORTABLE +
-            ' ' +
-            (sortOrder[sortOrder.length - 1][1]
+
+        descAsc = sortOrder[sortOrder.length - 1][1]
                 ? CLASSNAME_DESC
-                : CLASSNAME_ASC)
+                : CLASSNAME_ASC
+
+        th.classList.add('CLASSNAME_SORTABLE')
```

Comment:
should be  "th.classList.add(CLASSNAME_SORTABLE)" as this is currently adding a string instead of the "sortable" class